front/next.config.js

```diff
@@ -45,7 +45,7 @@ module.exports = {
           {
             key: "Content-Security-Policy",
             value:
-              "frame-ancestors 'self' https://*.salesforce.com https://*.force.com;",
+              "frame-ancestors 'self' https://*.salesforce.com https://*.force.com chrome-extension://ijppfkjmlibdkjpiakcjbhdcofbamoni;",
```

Comment:
??

Reply:
extension origin for cors

Reply:
Why do we need this? We never load the app from the extension, right?

Reply:
we do api calls from the extension. but that's not needed